app/models/educator.rb

```diff
@@ -37,6 +37,12 @@ def as_json(options = {})
     super(options.merge({ except: [:student_searchbar_json] }))
   end
 
+  def lock_access!(opts = { })
```

Comment:
How does this actually get called within the sequence of Devise steps?  I'm wondering because I don't think we have an educator model until our strategy runs here: https://github.com/studentinsights/studentinsights/blob/master/app/lib/ldap_authenticatable_tiny/strategy.rb#L18

So I'm wondering how `lockable` would be able to interrupt in the middle of that strategy's code.  That seems like the place we'd want the check to happen - once we verify the email is valid and we load that educator model, we check whether they've been locked.  Since that isn't happening in `strategy.rb`, I'm wondering how lockable can influence what happens.